docker/sui-antithesis/docker-compose-antithesis.yaml

```diff
@@ -167,7 +167,7 @@ services:
       - PRIMARY_GAS_OWNER=0xd59d79516a4ed5b6825e80826c075a12bdd2759aaeb901df2f427f5f880c8f60
       - GENESIS_BLOB_PATH=/opt/sui/config/genesis.blob
       - KEYSTORE_PATH=/opt/sui/config/sui.keystore
-      - STRESS_TARGET_QPS=3
+      - STRESS_TARGET_QPS=10
```

Comment:
How about setting STRESS_TARGET_QPS=11 STRESS_SHARED_COUNTER=5 STRESS_TRANSFER_OBJECT=5 BATCH_PAYMENT=1, or something like that to not scale up batch payment workload as much?